lib/components/commentsBox/commentsBox.less

```diff
@@ -64,6 +64,8 @@
     }
 
     .img-avatar {
+        width: 50px;
```

Comment:
With the current CSS, the Gravatars were rendering way too big in Firefox.  This CSS changed fixed the issue for me.  Sorry for mixing it in with the un-related admin hook.